### PR TITLE
Fix gateway-bitcoin benchmarks

### PR DESCRIPTION
**Pull request by koushiro**
Signed-off-by: koushiro <koushiro.cqx@gmail.com>

### CODE REVIEW

**Review by liuchengxu — CHANGES_REQUESTED** [review]
Please also update the old `weight_info.rs` https://github.com/chainx-org/ChainX/blob/develop-2.0/xpallets/gateway/bitcoin/src/weight_info.rs.

File: xpallets/assets/src/weights.rs [hunk 1]
```diff
@@ -40,22 +40,22 @@ pub trait WeightInfo {
...
             .saturating_add(T::DbWeight::get().reads(8 as Weight))
             .saturating_add(T::DbWeight::get().writes(6 as Weight))
     }
     fn force_transfer() -> Weight {
-        (288_613_000 as Weight)
+        (260_192_000 as Weight)
             .saturating_add(T::DbWeight::get().reads(8 as Weight))
             .saturating_add(T::DbWeight::get().writes(6 as Weight))
     }
     fn set_balance(_n: u32) -> Weight {
```
Comment (koushiro) [inline xpallets/assets/src/weights.rs:52]: @liuchengxu Do we need the parameter `n`?

**Comment by koushiro** [comment]
@liuchengxu PTAL

File: scripts/generate_weights.sh [hunk 2]
```diff
@@ -29,7 +29,7 @@ bench_run() {
 bench_run xpallet_assets            ./xpallets/assets/src/weights.rs
 bench_run xpallet_assets_registrar  ./xpallets/assets-registrar/src/weights.rs
 bench_run xpallet_dex_spot          ./xpallets/dex/spot/src/weights.rs
-# bench_run xpallet_gateway_bitcoin   ./xpallets/gateway/bitcoin/src/weights.rs
+bench_run xpallet_gateway_bitcoin   ./xpallets/gateway/bitcoin/src/weights.rs
 #bench_run xpallet_gateway_common    ./xpallets/gateway/common/src/weights.rs
```
Comment (liuchengxu) [inline scripts/generate_weights.sh:33]: Are you going to fix `common` benchmarks in the follow-up PR?

File: xpallets/assets/src/weights.rs (hunk 1 above)
Comment (liuchengxu) [reply xpallets/assets/src/weights.rs:52]: No, it's unused.

File: scripts/generate_weights.sh (hunk 2 above)
Comment (koushiro) [reply scripts/generate_weights.sh:33]: Yeah, I will try, but there is no guarantee that it will be fixed before the estimated time of 2.0 :(

**Review by liuchengxu — APPROVED** [review]

**Pull request merged**